miio/vacuum.py

```diff
@@ -93,10 +100,18 @@ class WaterFlow(enum.Enum):
     Maximum = 203
 
 
+class MopMode(enum.Enum):
+    """Mop routing on S7."""
+
+    Standard = 300
+    Deep = 301
```

Comment:
Is this maybe similar to https://github.com/rytilahti/python-miio/blob/master/miio/viomivacuum.py#L251 ? I'm trying to gather what features are supported by different vacuums to make it possible to provide an unified vacuum interface for downstream users.

Reply:
I think so. In deep mode the vacuum follows a tighter pattern for more intense cleaning while mopping. The viomi uses different patterns (Y and S) but in general it changes the routing of the robots so i guess you can call it similar.

Should I adapt my PR to the naming scheme based on the viomi?

Reply:
Hmm, I think it's okay to leave it as it is here now. Standard and deep sound more generic than S & Y, so when a common API for all vacuum devices is added, the former would be the likely candidates.